Sample Apps/Storefront/Storefront/CollectionCell.swift

```diff
@@ -114,7 +114,7 @@ class CollectionCell: UITableViewCell, ViewModelConfigurable {
 extension CollectionCell: StorefrontCollectionViewDelegate {
     
     func collectionViewShouldBeginPaging(_ collectionView: StorefrontCollectionView) -> Bool {
-        print("Products have should begin paging: \(String(describing: self.viewModel?.products.hasNextPage))")
+        print("Products should have begun paging: \(String(describing: self.viewModel?.products.hasNextPage))")
```

Comment:
I think this was supposed to say "Products should begin paging", to match the delegate callback.